ga4gh/backend.py

```diff
@@ -90,19 +91,18 @@ def runSearchRequest(
             request.pageSize = self._defaultPageSize
         if request.pageSize <= 0:
             raise exceptions.BadPageSizeException(request.pageSize)
-        pageList = []
+        response = protocol.SearchResponseBuffer(
```

Comment:
@jeromekelleher Actually your suggestion gets to the core of my confusion. I'd have nothing against that, except that the new `protocol.SearchResponseB*er` doesn't take a `responseClass` argument (explicitly or implicitly, as far as I can tell).
Which makes me wonder... why do we even have that class in the first place?